main.js

```diff
@@ -266,7 +267,7 @@ app = {
 			url: url
 		}).done((data) => {
 			const word = String(data["ParsedResults"][0]["ParsedText"]).trim().toLowerCase().split(" ").join("");
-			if (word.length > 3) {
+			if (word.length > 2) {
```

Comment:
why just this ? the bot is not work with your actual code try my code and if you don't like it i will give you the right to take onli this without credit me in code